src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  user: User | null,
+  todo: Todo | null,
+  onCloseBtn: () => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ user, todo, onCloseBtn }) => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {!user ? (
```

Comment:
what if there is `todo` but there is no `user`? here it would be better to add a check for `todo`. On line 50 you can check if `user` is not `null`
```suggestion
      {!todo ? (
```

Reply:
![image](https://user-images.githubusercontent.com/96202644/220706544-89dfe4c3-27d1-444a-9200-d156d1943802.png)
No, I cant do in way that you wish to be

Reply:
It must be direct in that place. not todo